utils_cv/tracking/model.py

```diff
@@ -281,47 +280,80 @@ def save(self, path) -> None:
 
         save_model(path, self.epoch, self.model, self.optimizer)
         print(f"Model saved to {path}")
-
-    def evaluate(self,
-                 results: Dict[int, List[TrackingBbox]],
-                 gt_root_path: str) -> str:
+                
+    def savetxt_results(self,
```

Comment:
Model.py in staging is different from the model.py in my initial commit in PR 566 (is that the one you used?), with updates following Patrick's comments. For e.g. savetxt_results was renamed to boxes_to_mot and moved to dataset.py

Reply:
Nvm, I see what you are doing. Is there any reason why you broke down the evaluate function into savetxt_results() + evaluate_mot() + mot_summary() and expose those to the user in the NB? Wouldn't we want the user to see as little as possible as what's going on under the hood? @PatrickBue, what do you think is best?

Reply:
Yes. I separated @karineip's original evaluate() function. This one can not only provide to save tracking results into tmp, but also can save results into user provided path. The reason I change is that when running 02 notebook, it's better to save tracking results and user doesn't have to re-run the prediction if the results are already saved in some paths. The mot challenge data is large and don't have to run every time.